MapboxMobileEvents/MMEMetricsManager.m

```diff
@@ -0,0 +1,143 @@
+#import "MMEMetricsManager.h"
+#import "MMEEvent.h"
+#import "MMEReachability.h"
+#import "MMEConstants.h"
+
+@interface MMEMetricsManager ()
+
+@property (nonatomic) MMEMetrics *metrics;
+
+@end
+
+@implementation MMEMetricsManager
+
++ (instancetype)sharedManager {
+    static MMEMetricsManager *_sharedManager;
+    static dispatch_once_t onceToken;
+    
+    dispatch_once(&onceToken, ^{
+        _sharedManager = [[MMEMetricsManager alloc] init];
+        _sharedManager.metrics = [[MMEMetrics alloc] init];
```

Comment:
Looks like you're already allocating a `MMEMetrics` below - is this necessary here?

Reply:
It shouldn't be necessary. I had added the `[super init]` below for testing and forgot I had this up here.